content/assignments/memory-allocator/src/helpers.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#pragma once
+
+#include <errno.h>
+#include <limits.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#define DIE(assertion, call_description)						\
+	do {														\
```

Comment:
Make those backslashes align. It looks ugly otherwise.

Reply:
"Fixing it" will make it look weird locally if you use a tab size of 4. If you are alright with this I will make it prettier on github.

Reply:
Tab size is **always** 8. Fix your damn settings.